pombo.py

```diff
@@ -757,15 +769,17 @@ def work(self):
                         start = time.time()
                         self.snapshot(current_ip)
                         runtime = time.time() - start
+                    if self.stolen(): 
                         time.sleep(wait_stolen - runtime)
                     else:
-                        time.sleep(wait_normal)
+                        time.sleep(wait_normal - runtime)
             else:
                 current_ip = self.public_ip()
                 if current_ip and self.need_report(current_ip):
                     wait = 60 * self.configuration['time_limit'] // 3
-                    for i in range(1, 4):
-                        self.log.info('* Attempt %d/3 *', i)
+                    numRetries = 3 if self.stolen() else 1
```

Comment:
Ideally, variables are lower case with underscore as PEP8 asked and to follow current script usages. Could you change this please?